### PR TITLE
refactor(ado): Remove Fortify, add in Spotbugs and Semmle

### PR DESCRIPTION
**Pull request by timtay-microsoft**
As per the latest SDL guidance, we should run Spotbugs at the gate, and both Spotbugs and Semmle in our nightly builds

### CODE REVIEW

File: vsts/sdl.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ phases:
 
   condition: succeeded()
   queue:
-    name: cloud-hosted-sdl-java
```
Comment (timtay-microsoft) [inline vsts/sdl.yaml:13]: No longer need to run any of our SDL tasks on this self hosted agent :)

File: vsts/sdl.yaml [hunk 2]
```diff
@@ -34,26 +34,42 @@ phases:
...
+    ## with annotations like @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
+  - powershell: ./vsts/spotbugs.ps1
+    displayName: 'Run Spotbugs'
+
+    ## Semmle doesn't have a false positive list, but instead recommends marking the lines of code in question
+    ## with "// lgtm"
+    ## https://help.semmle.com/lgtm-enterprise/user/help/alert-suppression.html#suppresswarnings-annotation
+  - task: Semmle@1
+    #Only run this step if it isn't a pull request. Semmle only needs to run nightly
+    condition: ne(variables['Build.Reason'], 'PullRequest')
```
Comment (timtay-microsoft) [inline vsts/sdl.yaml:47]: SDL requirements are to run spotbugs at the gate, and both spotbugs and Semmle nightly

**Review by jamdavi — APPROVED** [review]

**Pull request merged**